### ISSUE TITLE
Duplicate Ores/Energy bug

### ISSUE THREAD

**Issue by Mageloop** [pahimar/Equivalent-Exchange-3#1073]
Hello there, 
I was playing the FTB Inventions pack when I found a way to duplicate Ore with a combination of the transmutation table and the Ender IO Sag Mill. The idea is simple. Create Gold Ore from the table and Sag Mill it. Smelt the dust and put them in the Table again. The Ore and Bar have the *same* value. Infinite energy!

Thanks for your time!

**Comment by HadesDurin** [comment]
which version of EE3 is used in the pack? 
latest 0.3.0.547?

**Comment by pahimar** [comment]
Currently (I just downloaded the pack) v1.0.0 of FTB Inventions is using v0.3.0.544 of EE3 which had a bug in the Blacklist system. This allowed players to learn ores when they shouldn't have been able to, and enables the exploit listed in the OP. This has been fixed for a few days now in the latest release, FTB just needs to update the version of EE3 in the pack (they are testing the latest release toady iirc).

TLDR - FTB Inventions pack update will include the version of EE3 that fixes this bug.
